data/campaigns/Eastern_Invasion/scenarios/15_Return_to_Wesnoth.cfg

```diff
@@ -0,0 +1,977 @@
+        [remove_item]
+        [/remove_item]
+        [remove_time_area]
+            id=
+        [/remove_time_area]
+        [time_area]
+            x,y=0-99,0-99
+            [time]
+                id=indoors
+                name= _ "Indoors"
+                image=misc/time-schedules/schedule-indoors.png
+                lawful_bonus=25
+            [/time]
+        [/time_area]
+        [replace_map]
+            map_file=Throne_Room.map
+            expand,shrink=yes,yes
+        [/replace_map]
+
+        {PLACE_IMAGE scenery/throne.png 10 5} # if I change these, also change 19_Epilogue
+        {PLACE_IMAGE items/sceptre-of-fire.png 7 6}
+        {PLACE_IMAGE items/ornate2.png 13 6}
+        {PLACE_IMAGE items/ornate1.png 14 6}
+
+        {GENERIC_UNIT 2 (Pikeman)    8 20} {FACING se}
+        {GENERIC_UNIT 2 (Swordsman) 12 20} {FACING sw}
+        {GENERIC_UNIT 2 (Iron Mauler)    6 15} {FACING ne}
+        {GENERIC_UNIT 2 (Iron Mauler)   14 15} {FACING nw}
+        {GENERIC_UNIT 2 (Royal Guard)    6 11} {FACING se}
+        {GENERIC_UNIT 2 (Royal Guard)   14 11} {FACING sw}
+
+        [unit]
+            side=2
+            x,y=10,7
+            facing=se
+            {CHARACTER_STATS_KING}
+            [modifications]
+                {KONRAD_UNARMED_SCENARIO}
+            [/modifications]
+        [/unit]
+        [unit]
+            id=Aeraeka
+            name= _ "Aeraeka"
+            type=Arch Mage
+            side=2
+            x,y=12,8
+            facing=sw
+            gender=female
+            [modifications]
+                {TEAM_COLOR_OVERRIDE () brown}
+            [/modifications]
+        [/unit]
+        [unit]
+            id=Halric
+            name=_ "Halric"
+            type=General
+            profile=portraits/humans/marshal.webp
+            side=2
+            x,y=8,8
+            facing=se
+            [modifications]
+                {TEAM_COLOR_OVERRIDE () green}
+            [/modifications]
+        [/unit]
+        {NAMED_UNIT 2 (Lancer) 9 10 "Denogyc" (_"Denogyc") ()} {FACING ne}
+
+        [delay]
+            time=1500
+        [/delay]
+
+        [scroll_to]
+            x,y,immediate=10,10,yes
+        [/scroll_to]
+        [delay]
+            time=1500
+        [/delay]
+
+        [screen_fade]
+            red,green,blue=0,0,0
+            alpha=0
+            duration=500
+        [/screen_fade]
+
+        [fire_event]
+            name=council_start
+        [/fire_event]
+    [/event]
+
+    #---------------------------
+    # PLAY COUNCIL CUTSCENE
+    #---------------------------
+    [event]
+        name=council_start
+        [lock_view]
+        [/lock_view]
+
+        # the council discusses the northern alliance
+        [message]
+            speaker=Aeraeka
+            message= _ "Well try again! It is absolutely imperative that we get a response from the northerners."
```

Comment:
Capitalise 'northerners'?

Reply:
Just noting in this particular instance, this is a reference to the Northern Alliance, not the dunefolk calling the Wesnothians 'northerners'.